### ISSUE TITLE
[BUG] Seagate 'Command Timeout' Field - Wrong Parsing

### ISSUE THREAD

**Issue by gabriel-gehrke** [AnalogJ/scrutiny#734]
See attached the screenshot below.
After starting the app (using Docker) I was surprised to immediately see one of my drives being reported as "failed".  For context, it is a Seagate Firecuda SSHD with 2 TB of storage (ST2000DX002-2DV164).

![grafik](https://github.com/user-attachments/assets/a53545a7-351c-46e0-9b5b-0bcf58d8c69c)

According to [this post](https://superuser.com/questions/1747844/interpreting-seagate-smart-value-188-command-timeout-data-format) on the superuser forum this field needs further interpretation and the given value is not indicative of failure, just because it exceeds the decimal value 100. In fact, the reported failure rate can be calculated to be `1 errors in 65.545 operations` according to [this calculator](https://www.disktuna.com/seagate-raw-smart-attributes-to-error-convertertest/#4295032841), which is significantly below the displayed threshold value of 10%.

**Comment by Drynael** [comment]
I stumbled across this "Failure" also Today:

![image](https://github.com/user-attachments/assets/34320a7b-5f0a-45f1-8118-3a84c9bb3ebb)

According the mentioned Website my Disk has only 2 errors in 131.080 operations.

**Comment by tomini** [comment]
Hello,
is it normal that from four drives that are the same model (ST16000NM001G-2KK103) two of them passed and the other two did not, because of the `188 (0xBC) Command Timeout`?
Thanks
![image](https://github.com/user-attachments/assets/85e6a06e-f5dc-4b28-9535-d7d53b4c64f3)
![image](https://github.com/user-attachments/assets/70a95e25-2502-408c-9821-14952182b8b9)

**Comment by sscheib** [comment]
Just encountered the same issue with two of my drives (Seagate IronWolf 12TB). Four of my drives in total (Seagate IronWolf 12TB) show this exact same warning, but two with the raw value "100" which apparently means "0 errors in 100 operations."

**Comment by sscheib** [comment]
I've created a small BASH script that parses SMART Value 188 (Command Timeout) based on my understanding of the [Seagate SMART Attribute Specification](http://t1.daumcdn.net/brunch/service/user/axm/file/zRYOdwPu3OMoKYmBOby1fEEQEbU.pdf).
The calculator posted earlier (https://www.disktuna.com/seagate-raw-smart-attributes-to-error-convertertest/#4295032841) does not calculate Command Timeout.

I've created a [gist](https://gist.github.com/sscheib/1cd8b15003e9c698554e3ed6aeee11f0) for that script if anybody wants to give it a try.

My HDDs reported this:

```
Device: /dev/sdnb
Command Timeout (SMART ID 188):
  Raw Value         : 4295032833 (Hex: 000100010001)
  Total Timeouts    : 1
  Commands over 5s  : 1
  Commands over 7.5s: 1
```

I'd love to contribute to this project, but I've never worked with GO.
While looking around in the source, I couldn't find a portion of the code that handles special cases (for specific vendors or similar - but I might have missed it), which leads me to believe that something like that would need to be implemented from scratch, which I am certainly not capable of doing :/

**Comment by sscheib** [comment]
After a little more research, I found that this issue had been discussed earlier, see: https://github.com/AnalogJ/scrutiny/issues/522

TL;DR: Not an issue of Scrutiny, but of smartmontools.
The PR is currently pending upstream: https://github.com/smartmontools/smartmontools/pull/247

To have a fix right now, the following adjustment can be done to the `collector.yaml` to instruct `smartctl` to use the correct CLI values:

```
 - device: /dev/sdN
   type: 'sat'
   commands:
      metrics_smart_args: '-xv 188,raw16 --xall --json -T permissive'
```

The above `-xv 188,raw16` instructs `smartctl` to parse SMART value 188 as three 16-bit unsigned base-10 integers (see man: https://linux.die.net/man/8/smartctl) which results in the correct values.

How to apply:
1. Adjust `collectors.yaml` with the above `commands` option
2. Delete the failing HDDs in the Scrutiny WebUI
3. Restart the container. The new values for the prior failing HDDs will now populate correctly

I think this issue can be closed.

**Comment by LostOnTheLine** [comment]
> After a little more research, I found that this issue had been discussed earlier, see: [#522](https://github.com/AnalogJ/scrutiny/issues/522)
> 
> TL;DR: Not an issue of Scrutiny, but of smartmontools. The PR is currently pending upstream: [smartmontools/smartmontools#247](https://github.com/smartmontools/smartmontools/pull/247)
> 
> To have a fix right now, the following adjustment can be done to the `collector.yaml` to instruct `smartctl` to use the correct CLI values:
> 
> ```
>  - device: /dev/sdN
>    type: 'sat'
>    commands:
>       metrics_smart_args: '-xv 188,raw16 --xall --json -T permissive'
> ```
> 
> The above `-xv 188,raw16` instructs `smartctl` to parse SMART value 188 as three 16-bit unsigned base-10 integers (see man: https://linux.die.net/man/8/smartctl) which results in the correct values.
> 
> How to apply:
> 
>     1. Adjust `collectors.yaml` with the above `commands` option
> 
>     2. Delete the failing HDDs in the Scrutiny WebUI
> 
>     3. Restart the container. The new values for the prior failing HDDs will now populate correctly
> 
> 
> I think this issue can be closed.

I did this & my drive went back to being `Healthy` but after a couple days it shows as `Failed` again. `CrystalDiskInfo` shows the `Command Timeout` value as `1` but `Scrutiny` shows it as `290`
